java/src/test/java/com/saucelabs/simplesauce/MacOsTest.java

```diff
@@ -47,15 +47,6 @@ public void defaultSafari_browserVersionIs12_0() {
         assertEquals("latest", safariVersionSetThroughSauceSession);
     }
 
-    @Test
-    public void defaultSafari_macOsVersionIsMojave() {
```

Comment:
Why did you choose to remove this test?

Reply:
The test is removed because the functionality is removed because the code needs to be able to pass these tests: https://github.com/saucelabs/simple_sauce/pull/104/files#diff-6a67e96d9a405e84d935eea01fd071efR87-R103